modules/core/shared/src/main/scala/com/magine/http4s/aws/CredentialsProvider.scala

```
@@ -156,35 +162,24 @@ object CredentialsProvider {
         }
       } yield uri
 
-    def readTokenFile: F[Option[Header.Raw]] = {
-      def readFile(path: Path): F[String] =
-        Files
-          .forAsync[F]
-          .readAll(path)
-          .through(utf8.decode)
-          .compile
-          .string
-          .adaptError { case _: NoSuchFileException => MissingCredentials() }
-
-      ContainerAuthorizationTokenFile[F].read
-        .flatMap[Option[Header.Raw]] {
-          case Some(path) =>
-            for {
-              s <- readFile(path)
-            } yield Some(Header.Raw(ci"Authorization", s.trim))
-          case None => none[Header.Raw].pure[F]
-        }
-        .recover { case _ => none[Header.Raw] }
```

Similar to above, we do not want to mask errors here.